src/Commands/AzureAD/RegisterAzureADApp.cs

```diff
@@ -637,5 +645,52 @@ private void StartConsentFlow(string loginEndPoint, AzureADApp azureApp, string
                 WriteObject(record);
             }
         }
+
+        private void SetLogo(AzureADApp azureApp, string token)
+        {
+            if (!Path.IsPathRooted(LogoFilePath))
+            {
+                LogoFilePath = Path.Combine(SessionState.Path.CurrentFileSystemLocation.Path, LogoFilePath);
+            }
+            if (File.Exists(LogoFilePath))
+            {
+                try
+                {
+                    WriteVerbose("Setting the logo for the Azure AD app");
+
+                    var endpoint = $"https://{AuthenticationManager.GetGraphEndPoint(AzureEnvironment)}/v1.0/applications/{azureApp.Id}/logo";
+
+                    var bytes = File.ReadAllBytes(LogoFilePath);
+                    var byteArrayContent = new ByteArrayContent(bytes);
+
+                    var mimeType = "";
+                    if (LogoFilePath.EndsWith("gif", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Wouldn't it be an idea to run the path through a FileInfo instance so you can use the .Extension attribute, which would be a safer/cleaner approach, in my opinion?